main.js

```diff
@@ -399,6 +402,26 @@ app.whenReady().then(() => {
     widget.mainWindow.webContents.send(`set-calc-trend`, calcTrend, isMMOL);
   });
 
+  ipcMain.handle(`toggle-always-on-top`, () => {
+    const newState = !widget.mainWindow.isAlwaysOnTop();
+    widget.mainWindow.setAlwaysOnTop(newState);
+    return newState;
+  });
+
+  ipcMain.handle(`is-always-on-top`, () => {
+    return widget.mainWindow.isAlwaysOnTop();
+  });
+
+  if(config.get(`SHORTCUT.ENABLED`)) {
+    const ret = globalShortcut.register(config.get(`SHORTCUT.COMBINATION`) , () => {
```

Comment:
This implementation has a few issues:
	1.	Functionality: On macOS, the Alt key is not present on standard keyboards, so users can’t use the default key combination.
	2.	Usability: A better practice is to use the key.code handler to detect key presses when registering key combinations. This approach ensures consistency across different operating systems in a cross-platform application. Without this, you’ll need to handle many platform-specific differences and could encounter erroneous user inputs that might lead to unexpected app behavior or crashes.